### PR TITLE
Remove includes of removed helpers

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb Ready for review.

File: spec/features/checkout_spec.rb [hunk 1]
```diff
@@ -142,7 +142,7 @@
       # Need to do this now because the token stored in the DB is the encrypted version
       # The 'plain-text' version is sent in the email and there's one way to get that!
       reset_password_email = ActionMailer::Base.deliveries.first
-      token_url_regex = /^http:\/\/www.example.com\/user\/spree_user\/password\/edit\?reset_password_token=(.*)$/
+      token_url_regex = %r{^http://#{Regexp.escape(store.url)}/user/spree_user/password/edit\?reset_password_token=(.*)$}
```
Comment (dkubb) [inline spec/features/checkout_spec.rb]: Should this use the stronger anchors `\A` and `\z` ?

File: spec/features/checkout_spec.rb [hunk 2]
```diff
@@ -1,15 +1,15 @@
 RSpec.feature 'Checkout', :js, type: :feature do
-  given!(:store) { create(:store) }
-  given!(:country) { create(:country, name: 'United States', states_required: true) }
-  given!(:state)   { create(:state, name: 'Maryland', country: country) }
+  given!(:store)   { create(:store, default: true)                                             }
+  given!(:country) { create(:country, iso: 'us', name: 'United States', states_required: true) }
```
Comment (dkubb) [inline spec/features/checkout_spec.rb]: Not that it affects the output, the ISO alpha 2 codes are typically uppercased like `'US'`.

File: spec/features/checkout_spec.rb (hunk 1 above)
Comment (mbj) [inline spec/features/checkout_spec.rb]: No, because the email contains multiple lines.

File: spec/features/checkout_spec.rb (hunk 2 above)
Comment (mbj) [inline spec/features/checkout_spec.rb]: Actually it should have raised when I tried to use a non valid ISO code. For now I'll comply and use uppercase, good catch.

File: spec/controllers/spree/user_sessions_controller_spec.rb [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
 
-  let(:user) { create(:user) }
+  let(:user)       { create(:user) }
+  let(:other_user) { create(:user) }
 
   before { @request.env['devise.mapping'] = Devise.mappings[:spree_user] }
 
   context "#create" do
     context "using correct login information" do
+      let(:guest_token) { '0123456789012345' }
```
Comment (dkubb) [inline spec/controllers/spree/user_sessions_controller_spec.rb]: Do you mind adding another blank line after this? it'll make it a bit more readable.

Comment (mbj) [inline spec/controllers/spree/user_sessions_controller_spec.rb]: Good catch, also I'll change the literal to `SecureRandom.hex(16)` as the value has no meaning, but identity.

Comment (mbj) [inline spec/controllers/spree/user_sessions_controller_spec.rb]: Actually we should use length `24` as this is the latest format used.

**Comment by mbj** [comment]
@dkubb Ready for the next pass.

**Pull request merged**